Bottom: 052906ef9bb9474dc27a2a54dc179a41af1a4e43
Top:    ac757b592b1e6633a04403b18f3f3981bd7c6981
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-04 14:53:38 +0000

Refresh of rename-v-e-and-e-v-to-clairfy

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 96655be..3d55e94 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1633,7 +1633,7 @@ Gather (VNODE *startv, PLINE **result, J_Rule j_rule, DIRECTION initdir)
 	  poly_InclVertex (PREV_VERTEX (&(*result)->head), newn);
 	}
 #ifdef DEBUG_GATHER
-      DEBUGP ("gather vertex at %#mD\n", cur->point[0], cur->point[1]);
+      DEBUGP ("gather vertex at %#mD\n", curv->point[0], curv->point[1]);
 #endif
       /* Now mark the edge as included.  */
       newn = (dir == FORW) ? VERTEX_FORWARD_EDGE (curv) : VERTEX_BACKWARD_EDGE (curv);
